ledger/ledger-configuration/src/test/suite/scala/com/daml/ledger/configuration/ConfigurationSpec.scala

```diff
@@ -0,0 +1,244 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
```

Comment:
I'd suggest adding a test case for an unknown serialization version as well.

Reply:
Good idea. I added one for a version 3, which should also prompt anyone adding a v3 to also add tests. :slightly_smiling_face: